### ISSUE TITLE
Can I throw an exception without opening the circuit breaker?

### ISSUE THREAD

**Issue by guilhermednt** [eljam/circuit-breaker#9]
tl;dr: add a list of "allowed exceptions" that will be thrown without opening the circuit.

I'm using this lib to, among other things, protect the interaction with my SMS service.

This service eventually throws some validation exceptions like `InvalidCountryException` when the phone I try to text is not within the list of supported/allowed countries.

I'd like to catch this exception without opening the circuit since the service itself didn't actually fail.

Do you think this would make sense? If so I can arrange the PR so you don't have to bother with it.

BTW, do you have some way I could donate you a coffee?

Thanks for the lib!

**Comment by guilhermednt** [comment]
> If so I can arrange the PR so you don't have to bother with it.

Scratch that... PR #10 is already here for you to check. :)

**Comment by guilhermednt** [comment]
Thanks! I'm glad to help!

Closing since it was merged by PR #10
